static/js/web-install.js

```diff
@@ -7,6 +7,8 @@ const RELEASES_URL = "https://releases.grapheneos.org";
 const CACHE_DB_NAME = "BlobStore";
 const CACHE_DB_VERSION = 1;
 
+var safeToLeave = true;
```

Comment:
This should be `let` instead of `var`. No `var` should be used in our code. It's a legacy way of declaring variables.

Reply:
Ideally this would be checked by `eslint`. It might be possible.

Reply:
Opening additional pull for eslint changes. Also resolved.

Reply:
See: #202